Add a README, formatting guide, make FightingAnimation an animation library, add Gradle...

gamemanj opened the pull request:
(EDIT: Quite a bit now)

20kdc committed:
Remove NOTE

It's not needed anymore.

20kdc committed:
Create a README.

I'll do Gradle, followed by License, tomorrow if I can.

ljfa-ag commented:
The formatting is messed up, need either two trailing spaces or a double newline to get a newline in Markdown.

20kdc committed:
Added Gradle (with some adjustments to make it Java 6 compatible)

20kdc committed:
How'd this not get put in the previous commit?

20kdc committed:
Added gradle wrapper, removed eclipse projectfile
The Eclipse project file shouldn't be needed anymore.
The gradle wrapper is because that's the standard way to set up a project.

20kdc committed:
Ran everything through formatter, added a mini-styleguide
Hope it's not too horrible!
I know there's an option in Eclipse for this... somewhere...

20kdc committed:
Removed PMXTransformThreadPool from PMXModel, and made model visible again
Vic, you may have forgotten that transformCore adds the vertices to a FloatBuffer.
That has been resolved now. Also, PMXTransformThreadPool is still around (for reference)

20kdc committed:
Removed a useless emote ("gendo")

20kdc committed:
Moved FightingAnimation over to the Library system
TODO: work out a way of making "use" auto-reset itself.

20kdc committed:
Merge branch 'master' of https://github.com/gamemanj/Instrumentality

20kdc committed:
Made adjustments to README (see ljfa's comment on my PR)

ghost commented:
ljfa: It's fixed now. I should start using topic branches for this kind of massive-edit-push I've just done, but since AFAIK Vic isn't up yet...

20kdc committed:
More README edits

ghost commented:
Vic gave permission over IRC, so...
